src/web/mjs/engine/BookmarkManager.mjs

```diff
@@ -145,7 +146,7 @@ export default class BookmarkManager extends EventTarget {
      * Callback will be provided with an error (or null if no error).
      */
     saveProfile( profile, callback ) {
```

Comment:
Will bookmarks be saved after changing the directory in settings?
If not they may be lost when the directory is changed and the application is closed right after.

If not it might be a good idea to save the bookmarks whenever the settings changed event is fired
Example: https://github.com/manga-download/hakuneko/blob/618ecde29a464e6a838fe03fb6c04738317f11d7/src/web/mjs/connectors/NineAnime.mjs#L46

Reply:
good point!
looking into it